docs/topics/governance-legal/index.md

```diff
@@ -67,11 +63,30 @@ As Central EGA is a Data Processor under GDPR definitions, all data submitters a
 
 Coming soon!
 
-## 3. Explore additional resources
+## 3. Complete steps to join Federated EGA
+
+The pathway to becoming a Federated EGA node includes governance/legal steps (red, top-right path) that must be completed:
+
+<img src="../../assets/img/Path-to-becoming-FEGA-member-v1.0.svg" alt="Colored diagram with icons and descriptions of technical and governance steps an interested Federated EGA node needs to complete to join the Network." width="1200"  align="right" />
+
+* **Contact FEGA with intent to join**. This step can be accomplished by engaging in the ELIXIR FHD Community ("[How do I start?](https://ega-archive.github.io/FEGA-onboarding/#how-do-i-start)") or by contacting Mallory Freeberg or Amy Curwin.
```

Comment:
Should we include here both of your emails as hyperlinks or something? Or between brackets at least.

Reply:
I wanted to avoid having my email on this site if I could. What we need to do is create a mailing list, probably, that redirects to myself, Amy, maybe others in the future. Our emails can be found pretty easily online, so I don't think it's a major issue to not include them here.